source/CargoHold.cpp

```diff
@@ -237,7 +243,9 @@ bool CargoHold::HasMissionCargo() const
 // Check if there is anything in this cargo hold (including passengers).
 bool CargoHold::IsEmpty() const
 {
-	return commodities.empty() && outfits.empty() && missionCargo.empty() && passengers.empty();
+	// The outfits map's entries are not erased if they are equal to zero, so
+	// it's not enough to just test outfits.empty().
+	return commodities.empty() && !HasOutfits() && missionCargo.empty() && passengers.empty();
```

Comment:
Shouldn't we not be able to call commodities.empty() for the same reason that we can't call outfits.empty()?

Reply:
Yeah I noticed that too. I was planning on making a follow up PR, or should I do it in this one?